data_models/okta_data_model.py

```diff
@@ -25,3 +26,10 @@ def get_event_type(event):
     if event.get("eventType") == "system.mfa.factor.deactivate":
         return event_type.ADMIN_MFA_DISABLED
     return None
+
+
+def get_actor_user(event):
+    actor = deep_get(event, "actor", "displayName")
+    if actor == "unknown":
+        actor = deep_get(event, "actor", "alternateId")
```

Comment:
Good point, are you thinking just in line 34? And, to your knowledge, is it possible for `displayName` to not be known, but it is ingested into Panther as a different value from `"unknown"`?

Reply:
I've not seen that situation but it seems at least possible.  Finding all the potential values of log fields is hard to do with just about any source I've encountred.

We could change 32 as well, and make its default 'unknown'. That way if 32 returned `null` instead of `unknown` for some reason it would still fall into the second check logic on 34.

Reply:
Sounds good to me 👍